packages/twenty-server/src/database/typeorm/core/migrations/common/1757364323932-1757000000000-addApplicationEntityAndRelationships.ts

```diff
@@ -0,0 +1,192 @@
+import { type MigrationInterface, type QueryRunner } from 'typeorm';
+
+export class AddApplicationEntityAndRelationships1757364323932
+  implements MigrationInterface
+{
+  name = 'AddApplicationEntityAndRelationships1757364323932';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+            DO $$ BEGIN
+                CREATE TYPE "core"."application_sourcetype_enum" AS ENUM('git', 'local', 'marketplace');
+            EXCEPTION
+                WHEN duplicate_object THEN null;
+            END $$;
+        `);
+
+    await queryRunner.query(`
+            CREATE TABLE IF NOT EXISTS "core"."application" (
+                "id" uuid NOT NULL DEFAULT uuid_generate_v4(),
+                "universalIdentifier" uuid NOT NULL,
+                "label" character varying NOT NULL,
+                "description" text,
+                "icon" character varying,
+                "repositoryUrl" character varying,
+                "version" character varying,
+                "sourceType" "core"."application_sourcetype_enum" NOT NULL,
+                "sourcePath" character varying,
+                "workspaceId" uuid NOT NULL,
+                "createdAt" TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
+                "updatedAt" TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
+                "deletedAt" TIMESTAMP WITH TIME ZONE,
+                CONSTRAINT "PK_569e0c3e863ebdf5f2408ee1670" PRIMARY KEY ("id"),
+                CONSTRAINT "CHK_APPLICATION_GIT_REPOSITORY_URL" CHECK (("sourceType" != 'git') OR ("repositoryUrl" IS NOT NULL)),
+                CONSTRAINT "CHK_APPLICATION_LOCAL_SOURCE_PATH" CHECK (("sourceType" != 'local') OR ("sourcePath" IS NOT NULL))
+            )
+        `);
+
+    await queryRunner.query(`
+            CREATE INDEX "IDX_APPLICATION_WORKSPACE_ID" ON "core"."application" ("workspaceId")
+        `);
+
+    await queryRunner.query(`
+            CREATE UNIQUE INDEX "IDX_APPLICATION_LABEL_WORKSPACE_ID_UNIQUE" ON "core"."application" ("label", "workspaceId") WHERE "deletedAt" IS NULL
+        `);
+
+    await queryRunner.query(`
+            ALTER TABLE "core"."application" ADD CONSTRAINT "FK_APPLICATION_WORKSPACE_ID" FOREIGN KEY ("workspaceId") REFERENCES "core"."workspace"("id") ON DELETE CASCADE
+        `);
+
+    await queryRunner.query(`
+            ALTER TABLE "core"."agent" ADD "applicationId" uuid
```

Comment:
All these relationships declaration were removed to do a simpler PR, please regenerate migration from scratch with `npx nx run twenty-server:typeorm migration:generate src/database/typeorm/core/migrations/[name] -d src/database/typeorm/core/core.datasource.ts`